app/assets/stylesheets/administrate/components/_pagination.scss

```diff
@@ -15,3 +15,15 @@
     font-weight: $bold-font-weight;
   }
 }
+
+.pagination-with-total {
+  margin-bottom: $base-spacing * 4;
+
+  .pagination {
```

Comment:
Is this needed? Can we simply reuse the `margin-bottom` already applied to `.pagination` on Line 1?